src/Xamarin.Android.NamingCustomAttributes/System.Runtime.CompilerServices/UnconditionalSuppressMessageAttribute.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
If this file was copied from elsewhere -- which I think it was -- then it should instead be in e.g. `src-ThirdParty/…`.

This was overlooked with `DynamicallyAccessedMemberTypes.cs` and `DynamicDependencyAttribute.cs`, so those should likewise move. :-(

Reply:
These should be moved around now.